spec/lib/bb/client_spec.rb

```diff
@@ -65,6 +65,6 @@
   it 'gets a text version of a report', :vcr do
     client_response = client.get_download_report('txt')
     expect(client_response.response_headers['content-type'])
-      .to eq('text/plain; charset=UTF-8')
+      .to eq('text/plain')
```

Comment:
This was the only real semantic change in the results - MHV no longer returning a charset parameter here. Didn't seem worth raising to MHV but if others disagree I can do so.